gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+networkTimeout=10000
```

Comment:
Is this needed?  Looks like the default is 10000?  https://docs.gradle.org/7.6/release-notes.html

Reply:
@mbcoder Good question. `networkTimeout` and `validateDistributionUrl` were added when I updated the gradle version by running `./gradlew wrapper --gradle-version 8.5`. Looking at the various `gradle-wrapper.properties` files in our java development repo, they all have:
```
networkTimeout=10000
validateDistributionUrl=true
```
It looks like `networkTimeout=10000` came in with Gradle version 7.6 and `validateDistributionUrl=true` came in with 8.2.

In light of that, would you agree it makes sense to leave them in?